conftest.py

```diff
@@ -1,5 +1,6 @@
 """Pytest configuration file."""
 
 pytest_plugins = [
+    "tests.fixtures.integration",
```

Comment:
So neat, so organised! We should definitely switch to grouping the fictures like that in `movement`.